dobson/main.py

```diff
@@ -7,34 +7,12 @@
 # snmpwalk -v1 -c public 192.168.0.1 iso.3.6.1.2.1.3.1.1.2.12.1
 
 import csv
-import subprocess
 import sys
 import time
-import dobson.constants as constants
 
 from slackclient import SlackClient
 
-
-def get_devices():
-    """Get the list of MAC addresses of connected devices
-
-    TODO: Make this method less specific to this model of router (TP-Link Archer C50)
-    """
-    output = subprocess.check_output(('/usr/bin/snmpwalk', '-v1', '-c', 'public', '192.168.0.1', 'iso.3.6.1.2.1.3.1.1.2.12.1'))
-    devices = output.decode('utf-8').rstrip().split('\n')
-
-    for device in devices:
-        # The output from SNMP looks like this, so we parse it for the IP and MAC address:
-        # iso.3.6.1.2.1.3.1.1.2.12.1.192.168.0.100 = Hex-STRING: 74 4A A4 CC 81 A1
-
-        mac_addr = device[-18:].strip(' ').lower().replace(' ', ':')
-        ip_addr = device[27:40]
```

Comment:
Ah, yeah I don't think this was actually used anywhere in the end, so thanks for removing it. Maybe it would be useful in the future sometime, but not yet.

Reply:
I removed the ip_addr part but the rest of the function was just split up and moved to utils (so it could be used in the `logusers.py` file)